### PR TITLE
Prevent tour from showing if already seen

### PR DESCRIPTION
**Pull request by LyubomirT**
# Issue Title: [Bug]: Tutorial appears even if the website has already been accessed before.
*closes* #434

## Type of change ☑️

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

 
## How Has This Been Tested? ⚙️

Hosted my own instance of the website and tested the new bugfix there, everything seemed to work.

### CODE REVIEW

**Comment by singodiyashubham87** [comment]
@LyubomirT You could have done the cookies part without using any library too, isn't it?

**Comment by LyubomirT** [comment]
@singodiyashubham87 Sorry for that, I just figured it'd be more convenient for you if I used JS-Cookie, I guess it's the opposite. Changing the code to work with cookies without it.

File: src/App.jsx
Comment (singodiyashubham87) [inline src/App.jsx:1]: Kindly add these functions related to cookies like setCookie & getCookie inside a separate file './src/utils/manageCookies.js" & import them in "App.jsx"

**Review by singodiyashubham87 — CHANGES_REQUESTED** [review]
Kindly do the addressed changes.

File: src/App.jsx
Comment (LyubomirT) [reply src/App.jsx:1]: Made changes, please review the code again and let me know if it works for you.

**Comment by LyubomirT** [comment]
@singodiyashubham87 Implemented what you asked for, please review the code again and let me know if it works well.

**Comment by singodiyashubham87** [comment]
@LyubomirT Kindly resolve the conflicts.

**Review by singodiyashubham87 — APPROVED** [review]
Approved!

**Comment by singodiyashubham87** [comment]
@LyubomirT Well Done!

**Pull request merged**